src/pages/whats-new.js

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import SEO from '../components/seo';
+import PageTitle from '../components/PageTitle';
+import { graphql } from 'gatsby';
+import { Layout, Link } from '@newrelic/gatsby-theme-newrelic';
+
+const MOBILE_BREAKPOINT = '960px';
+
+const WhatsNew = ({ data }) => {
+  const { allMarkdownRemark } = data;
+  const posts = allMarkdownRemark.edges.map(({ node }) => node);
+  const postsByDate = Array.from(
+    posts
+      .reduce((map, post) => {
+        const key = post.frontmatter.releaseDate;
+
+        return map.set(key, [...(map.get(key) || []), post]);
+      }, new Map())
+      .entries()
+  );
+
+  return (
+    <>
+      <SEO title="What's new in New Relic" />
+      <div>
+        <PageTitle
+          css={css`
+            margin-bottom: 2rem;
+          `}
+        >
+          What's new in New Relic
+        </PageTitle>
+        <Layout.Content>
+          <div
+            css={css`
+              --timeline-width: 4px;
+              --ring-size: 1rem;
+              --ring-border-width: var(--timeline-width);
+              --timeline-color: var(--color-neutrals-200);
+
+              display: grid;
+              grid-template-columns: auto 1fr;
+              grid-column-gap: 2rem;
+
+              .dark-mode & {
+                --timeline-color: var(--color-dark-200);
+              }
+
+              @media screen and (max-width: ${MOBILE_BREAKPOINT}) {
+                grid-template-columns: auto;
+              }
+            `}
+          >
+            {postsByDate.map(([date, posts], idx) => (
```

Comment:
I'm surprised React isn't yelling at you for not supplying a `key` 🤔

Reply:
Or maybe I was ignoring it 😛 

<img width="757" alt="Screen Shot 2020-12-09 at 9 44 28 AM" src="https://user-images.githubusercontent.com/565661/101666506-267d7080-3a03-11eb-8e6a-d4b39006161f.png">

Reply:
Good catch!

Reply:
Fixed in 43245ff